pkgs/by-name/re/redocly/package.nix

```diff
@@ -32,24 +32,27 @@ buildNpmPackage rec {
 
     mkdir $out/bin
     makeWrapper $out/lib/node_modules/@redocly/cli/node_modules/@redocly/cli/bin/cli.js \
-      $out/bin/redocly-cli \
+      $out/bin/redocly \
       --set-default REDOCLY_TELEMETRY off \
       --set-default CI true # Silence update messages
+
+    # Symlink for backwards compatibility. Remove after deprecation.
+    ln -s $out/bin/redocly $out/bin/redocly-cli
   '';
 
   installCheckPhase = ''
```

Comment:
Would be better to migrate to `passthru.tests` with `testers.testVersion`

Reply:
TIL, thanks!